cmd/blockchaincmd/add_validator.go

```diff
@@ -183,13 +176,13 @@ func addValidator(cmd *cobra.Command, args []string) error {
 	}
 
 	if len(args) == 0 {
-		if rpcURL == "" {
-			rpcURL, err = app.Prompt.CaptureURL("What is the RPC endpoint?", false)
+		if flags.RPC == "" {
```

Comment:
Is there anyway to refactor the flags to have an input argument to be required and for the prompt to be baked into the flag creation/adding to the cmd

Reply:
added this for rpc url, but only for addValidator. Will make another PR to extend it to other commands

Reply:
there may be some dependence between flag values, are flags that should only be used if some other flag value
is set. this will complicate automatic prompting unless we make something generic that takes this into account.

Reply:
this is outdated, alrdy addressed